src/readme-execution.md

```diff
@@ -0,0 +1,22 @@
+# Execution of the Weather-PWA
```

Comment:
@MiCh0r This is the doumentation for the execution steps. Test it first with the local http-server does it work on your machine?

Reply:
@planbsim The first step worked on my machine but i couldn't use `ng build` so i used `ionic build`. Is it ok?